bin/build-locales

```diff
@@ -15,65 +16,70 @@ const chalk = require('chalk');
 const toSource = require('tosource');
 const oneLine = require('common-tags').oneLine;
 
-const appName = config.get('appName');
+const app = config.get('appName');
+const buildAll = process.env.ADDONS_FRONTEND_BUILD_ALL === '1';
```

Comment:
Should this just be any value? `true` would be nicer to use on the command line... if that would ever get used locally?